src/content/en/2021/caching.md

```diff
@@ -17,4 +19,205 @@ featured_stat_3: TODO
+  sql_file="todo.sql"
+) }}
+
+Talk about `immutable` directive, 12% of desktop requests containing Cache-Control: immutable are on the "static.parastorage.com" host.
+
+Talk about invalid date, last modified and expires, (0.11% of dates in desktop requests are invalid).
+
+Desktop - Valid date strings:
+99.18% using date, 72.57% using last-modified, 55.48% using expires
+
+Mobile - Valid date strings:
+99.07% using date, 70.55% using last-modified, 55.57% using expires
+
+Although 99% of both Desktop and Mobile pages use valid date strings, only 55% of Desktop and 56% of Mobile pages use 'expires'. Last-modified is used at a higher percentage, at 73% for Desktop and 71% for Mobile. While the amount of invalid 'date' and 'last-modified' directives is less than 1%, 2.75% of Mobile and 3.20% of Desktop 'expires' are being used incorrectly.
+
+What is or would be the impact from using expires more effectively?
+
+Could we possibly see big wins from getting more consistent here?
+
+### Vary
+
+Talk about Vary adoption, 42% of requests on mobile set Vary: accept-encoding.
+
+The most common Vary headers are: `accept-encoding`, `user-agent`, `origin`, `accept`.
+
+### TTL adoption
+
+Talk about 28% of mobile requests use neither Cache-Control nor Expires headers.
+
+The median HTML resource on mobile has a TTL of 14 days
+
+## Cookie Cache adoption
+
+35.63% of Cacheable Desktop pages with set-cookie
+Mobile pages with set-cookie: 35.99%
+
+Another area of opportunity we discovered was with cacheable pages that are not using `set-cookie`. About 35% of Desktop and Mobile pages use `set-cookie` while over 64% of them do not.
+
+Large area of opportunity for pages that are actually cacheable, but are just missing this directive?
+
+Fun fact `test_cookie` is the most popular desktop and mobile cookie name at 4% of all cookies.
+
+{{ figure_markup(
+  caption="TODO",
+  content="4%",
+  classes="big-number",
+  sheets_gid="455647224",
+  sql_file="todo.sql"
+) }}
+
+The top cookies attributes used on Desktop and Mobile are SameSite and Secure. SameSite is on 53% of cookies set in Desktop responses and a whopping 68% on Mobile responses.
+
+Why is it a big deal that SameSite is at 68% on mobile?
+
+## What type of content are we caching?
+
+Talk about what kind of resources are we caching, (resource age groups: party, type).
+
+Talk about resource age, the median HTML resource on mobile is 9 weeks old.
+
+Talk about, total number of requests with comparable Last-Modified and expiration times, not necessarily all requests.
+
+Talk about 54% of mobile resources are older than their TTL.
+
+{{ figure_markup(
+  caption="TODO",
+  content="54%",
+  classes="big-number",
+  sheets_gid="768623684",
+  sql_file="todo.sql"
+) }}
+
+Total number of requests with comparable Last-Modified and expiration times, not necessarily all requests.
+
+Talk about, 60% of 1P mobile resources are older than their TTL, which is more common than 3P resources (45%).
+
```

Comment:
Explain why this is significant- 60% of 1st Party mobile resources are older than their TTL- is this because 1P are more important or crucial? Because 3P are being deprecated?